src/shared/messaging/port-finder.js

```diff
@@ -58,13 +59,16 @@ export class PortFinder {
       throw new Error('Invalid request of channel/port');
     }
 
+    const requestId = generateHexString(6);
```

Comment:
Something to think for the future: it may be valuable to return the requestId value together with the port in the promise of `PortFinder#discover`.

`const [sidebarPort, requestId] = await PortFinder.discover('sidebar')`

The `requestId` could be used to identify the provenance of certain RPC events, in cases of one to many connections: `1 sidebar : N guests` or `1 host: N guests`